pkg/values/packagemaps.go

```diff
@@ -702,6 +844,126 @@ var KernelPackagesModels = ModelPackageMap{
 				// 24.10 uses the 24.04 hwe kernel as it is the same hwe track https://ubuntu.com/kernel/lifecycle
 				">=24.10": {"linux-image-generic-hwe-24.04"},
 			},
+			AgxOrin: {
+				Common: {
+					// NVIDIA L4T packages for AGX Orin
+					"nvidia-l4t-3d-core",
+					"nvidia-l4t-apt-source",
+					"nvidia-l4t-bootloader",
+					"nvidia-l4t-camera",
+					"nvidia-l4t-configs",
+					"nvidia-l4t-core",
+					"nvidia-l4t-cuda",
+					"nvidia-l4t-display-kernel",
+					"nvidia-l4t-firmware",
+					"nvidia-l4t-gbm",
+					"nvidia-l4t-graphics-demos",
+					"nvidia-l4t-gstreamer",
+					"nvidia-l4t-init",
+					"nvidia-l4t-initrd",
+					"nvidia-l4t-jetson-io",
+					"nvidia-l4t-jetsonpower-gui-tools",
+					"nvidia-l4t-kernel",
+					"nvidia-l4t-kernel-dtbs",
+					"nvidia-l4t-kernel-headers",
+					"nvidia-l4t-kernel-oot-modules",
+					"nvidia-l4t-multimedia-utils",
+					"nvidia-l4t-multimedia",
+					"nvidia-l4t-nvfancontrol",
+					"nvidia-l4t-nvpmodel-gui-tools",
+					"nvidia-l4t-nvpmodel",
+					"nvidia-l4t-nvsci",
+					"nvidia-l4t-oem-config",
+					"nvidia-l4t-openwfd",
+					"nvidia-l4t-optee",
+					"nvidia-l4t-pva",
+					"nvidia-l4t-tools",
+					"nvidia-l4t-vulkan-sc-dev",
+					"nvidia-l4t-vulkan-sc-samples",
+					"nvidia-l4t-vulkan-sc-sdk",
+					"nvidia-l4t-vulkan-sc",
+					"nvidia-l4t-wayland",
+					"nvidia-l4t-weston",
+					"nvidia-l4t-x11",
+					"nvidia-l4t-xusb-firmware",
+					// CUDA packages
+					"cuda-cudart-12-6",
+					"cuda-cudart-dev-12-6",
+					"cuda-nvcc-12-6",
+					"cuda-nvdisasm-12-6",
+					"cuda-nvml-dev-12-6",
+					"cuda-nvprune-12-6",
+					"cuda-nvrtc-12-6",
+					"cuda-nvrtc-dev-12-6",
+					"cuda-nvtx-12-6",
+					"libcublas-12-6",
+					"libcublas-dev-12-6",
+					// Jetson GPIO
+					"jetson-gpio-common",
+					"python3-jetson-gpio",
+					// OpenCV
+					"libopencv-dev",
+				},
+			},
+			OrinNX: {
+				Common: {
```

Comment:
I think the packages are the same across all the L4T stack - would make sense to have a common piece for both the flavors (OrinNX and AGX Orin) to improve maintenance